proxylib/npds/client_test.go

```diff
@@ -94,6 +95,6 @@ func (s *ClientSuite) TestRequestAllResources(c *C) {
 	s.UpsertNetworkPolicy(c, xdsServer, resources[0])
 
 	time.Sleep(DialDelay * BackOffLimit)
-	c.Assert(s.acks, Equals, 1)
-	c.Assert(s.nacks, Equals, 0)
+	c.Assert(atomic.LoadUint64(&s.acks), Equals, 1)
```

Comment:
Travis failure is related - 

```
----------------------------------------------------------------------
FAIL: client_test.go:74: ClientSuite.TestRequestAllResources
client_test.go:98:
    c.Assert(atomic.LoadUint64(&s.acks), Equals, 1)
... obtained uint64 = 0x1
... expected int = 1
OOPS: 0 passed, 1 FAILED
--- FAIL: Test (11.00s)
```

```suggestion
	c.Assert(atomic.LoadUint64(&s.acks), Equals, uint64(1))
```

Reply:
Huh, thanks! Wonder why this didn't trigger when I tested locally. Maybe I tested the wrong package. Will first reproduce it locally and then fix.

Reply:
Turns out I wasn't running tests in the sub package. Fixed now.